src/js/base/handalone.js

```diff
@@ -21,7 +21,9 @@ requirejs(["pyret-base/js/runtime", "pyret-base/js/post-load-hooks", "pyret-base
 
   var runtime = runtimeLib.makeRuntime({
     stdout: function(s) { process.stdout.write(s); },
-    stderr: function(s) { process.stderr.write(s); }
+    stderr: function(s) { process.stderr.write(s); },
+    // TODO: this is semantically different. Is this ok?
+    stdin: process.stdin,
```

Comment:
I agree that's weird.  We need to take a careful look at _all_ usages of `std{in,out,err}` in the codebase -- some of them are directly using `process.std*`, some places call `runtime.stdout.write(...)`, and some places call `theOutsideWorld.stdout(...)`, so we clearly are inconsistent in usage and in types, already...  The intent of wrapping `stdout` and `stderr` was to encapsulate the Node-specific weirdness of whatever `process.stdout` and `process.stderr` are, so that we could more easily replace them with other implementations, and then we never really got around to it.

Reply:
Is it okay to move that careful look to a separate issue?

Reply:
Will open an issue and assign to myself once this PR is merged.